examples/todo/modules/shared-data/src/main/resources/logback.xml

```diff
@@ -0,0 +1,19 @@
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %highlight(%-5level) %cyan(%logger{15}) - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <root level="INFO">
+        <appender-ref ref="STDOUT" />
+    </root>
+
+    <logger name="org.tessellation" level="DEBUG" />
+    <logger name="xyz.kd5ujc" level="DEBUG" />
```

Comment:
?

Reply:
thank you, a relic from having ported this from a personal metagraph project